docs/sources/tutorials/get-started.md

```diff
@@ -1,11 +1,11 @@
 ---
 canonical: https://grafana.com/docs/alloy/latest/tutorials/get-started/
-description: Getting started with the tutorials
-title: Get started
+description: Getting started with Alloy
+title: Get started with Alloy
 weight: 10
 ---
 
-## Get started with {{% param "PRODUCT_NAME" %}}
+## Get started with {{< param "PRODUCT_NAME" >}}
```

Comment:
```suggestion
## Get started with {{% param "PRODUCT_NAME" %}}
```

We have to use `%` syntax in the headings.... so this change will have to be reverted.

The `%` controls when in the build process something is parsed, The `%` ensures the param shortcode is rendered before being passed to the markdown processor (which generates the .TableOfContent variable). If we use `<` and `>` for a shortcode in a heading, the variable substitution happens at the wrong stage. If we don't do this, Hugo inserts `hahahaha` in t he URL and anchors.

tl;dr: Use `%` in headings and `<` and `>` in body text.